### ISSUE TITLE
No entry points found for Windows DLL's.

### ISSUE THREAD

**Issue by ThadHouse** [wpilibsuite/ntcore#13]
I was able to get a successful build on a VS 2013 machine, but when trying to open the DLL it says it cannot find the entry point its looking for. Running dumpbin on the compiled DLL shows no functions at all being exported from the DLL. Are there any special gradle flags needed to get the exports to build correctly?

**Comment by PeterJohnson** [comment]
I've always used cmake to build on Windows.  The ntcore.def looks like it's being used in gradle too, but maybe something's broken there.  @333fred, help?

**Comment by ThadHouse** [comment]
The CMake build does work. I did find a typo in the ntcore.def file for the NT_PostRpcResponse function, which was NT_PostRpcRepsonse. I added that fix to the pull request, which fixed my cmake builds, but it did not fix the gradle build.

**Comment by 333fred** [comment]
Sorry for the lack of responses guys. I'm currently transferring planes,
I'll be able to look tomorrow. As to msvc 2015, gradle has some issues with
it and doesn't include the right headers, causing compilation issues.

The CMake build does work. I did find a typo in the ntcore.def file for the
NT_PostRpcResponse function, which was NT_PostRpcRepsonse. I added that fix
to the pull request, which fixed my cmake builds, but it did not fix the
gradle build.
